docs/reference/resources.md

```diff
@@ -2,6 +2,8 @@
 title: "Resources"
```

Comment:
```suggestion
title: "Other Resources"
```

Reply:
Clicking through the staged content i realized we have this "optional" type content up above the API when it should probably live at the bottom. I added "other" to try and clarify its relationship but open to other suggestions.

Reply:
Updated it to additional resources and set the position as 100 to put it lower, does that work? :slightly_smiling_face: